### DISCUSSION TITLE
I am on Ubuntu 22.04 and slurm 24.05 version how do i use the slurm-web?

### DISCUSSION THREAD

**Discussion by ghost** [rackslab/Slurm-web#338] [Q&A] [answered]
I am on Ubuntu 22.04 and slurm 24.05 version how do i use the slurm-web?

**Comment by rezib** [comment, marked as answer]
Hello @shrvi,

Unfortunately, regarding Ubuntu, we only support 24.04 LTS for the moment because the version of nodejs/npm (dependencies of Slurm-web) are a bit old on Ubuntu 22.04 LTS and some more work is required to be compatible with Ubuntu 22.04 LTS.

If you are interested, Rackslab can do the effort to build and maintain packages for Ubuntu 22.04 LTS but we will charge for this. Are you interested by a quotation?

Another option is to distribute docker/podman containers that would be compatible with all old distributions. We have issue #266 opened to track this feature and again, Rackslab is looking for a customer to fund this development. Tell me if you are interested.

**Comment by ghost** [comment]
Thanks, I will try on Ubuntu 24 then. Will it still work if the compute nodes are on Ubuntu 22 and slurm 24 version?

**Reply by rezib** [reply]
> Thanks, I will try on Ubuntu 24 then. Will it still work if the compute nodes are on Ubuntu 22 and slurm 24 version?

Yes, for sure. Ubuntu 24.04 LTS is required to install [Slurm-web Ubuntu packages](https://docs.rackslab.io/slurm-web/install/install/distribs/ubuntu.html) but there is no requirement for your compute nodes (which do not execute any Slurm-web component).
